app/views/content/is-teaching-right-for-me/become-an-early-years-teacher.md

```diff
@@ -26,14 +26,14 @@ calls_to_action:
   EY-graduate:
     name: feature_table
     arguments:
-      - Cost: "Your course fees of up to £7,000 are covered by the Department for Education and are paid directly to the training provider. You do not need to do anything to access this funding."
+      - Cost: "Your course fees of up to %{earlyyears_coursefeescoveredbydfe} are covered by the Department for Education and are paid directly to the training provider. You do not need to do anything to access this funding."
```

Comment:
@MylesJarvis You've probably seen this already but these figures don't render properly on the front end

Rather than look at a dev solution for this, we could look at it in terms of design - I don't _think_ we use the `feature_table` component anywhere else, so we could look to simplify our design components instead?

Reply:
Fixed.